website/docs/r/uplink_host_switch_profile.html.markdown

```diff
@@ -0,0 +1,130 @@
+---
+subcategory: "Fabric"
+layout: "nsxt"
+page_title: "NSXT: nsxt_uplink_host_switch_profile"
+description: A resource to configure a uplink host switch profile in NSX Policy.
+---
+
+# nsxt_uplink_host_switch_profile
+
+This resource provides a method for the management of a uplink host switch profile which can be used within NSX Policy.
+
+This resource is applicable to NSX Policy Manager.
+
+## Example Usage
+
+```hcl
+resource "nsxt_uplink_host_switch_profile" "uplink_host_switch_profile" {
+  description  = "Uplink host switch profile provisioned by Terraform"
+  display_name = "uplink_host_switch_profile"
+
+  mtu            = 1500
+  transport_vlan = 0
+  overlay_encap  = "GENEVE"
+  lag {
+    name                   = "t_created_lag"
+    load_balance_algorithm = "SRCDESTIPVLAN"
+    mode                   = "ACTIVE"
+    number_of_uplinks      = 2
+    timeout_type           = "SLOW"
+  }
+  teaming {
+    active {
+      uplink_name = "t1"
+      uplink_type = "PNIC"
+    }
+    standby {
+      uplink_name = "t2"
+      uplink_type = "LAG"
+    }
+    policy = "FAILOVER_ORDER"
+  }
+  named_teaming {
+    active {
+      uplink_name = "nt1"
+      uplink_type = "PNIC"
+    }
+    standby {
+      uplink_name = "nt2"
+      uplink_type = "PNIC"
+    }
+    policy = "FAILOVER_ORDER"
+    name   = "named_teaming1"
+  }
+  named_teaming {
+    active {
+      uplink_name = "nt3"
+      uplink_type = "PNIC"
+    }
+    policy = "LOADBALANCE_SRCID"
+    name   = "named_teaming2"
+  }
+
+  tag {
+    scope = "color"
+    tag   = "blue"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `display_name` - (Required) Display name of the resource.
+* `description` - (Optional) Description of the resource.
+* `tag` - (Optional) A list of scope + tag pairs to associate with this resource.
+* `nsx_id` - (Optional) The NSX ID of this resource. If set, this ID will be used to create the resource.
+* `mtu` - (Optional) Maximum Transmission Unit used for uplinks format: int32, minimum: 1280.
```

Comment:
in terraform we don't distinguish between int32 and int64, lets remove the format here and below

Reply:
Done.